hangman.py

```diff
@@ -50,8 +50,8 @@ def word_is_solved(word, guesses):
         bool: True if word is solved, False if word is not.
     """
 
-    for letter in word:
-        if letter not in guesses:
+    for letter in word.lower().strip():
```

Comment:
`.strip()` might not be the best here.

Reply:
We can use `"".join(word.lower().split())` since `split()` splits whitespace by default, but we begin to have readability issues. Another option is to import and use a regex module, but that is both excessive and significantly impacts performance (which may not be a huge concern with only one call per guess). I think the first option makes more sense, but feel free to give your input.

Reply:
Why are we using `split()` in the first place? What potential case is this solving?

Reply:
Perhaps 
```
if letter is space: 
    continue?
```